docs/installation-aws-ecs/aws-ecs-deployment.md

```diff
@@ -17,6 +17,12 @@ $ export AWS_PROFILE=profilename
 
 For the purpose of this instruction we used the [tutorial](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-cli-tutorial-ec2.html#ECS_CLI_tutorial_compose_create) on AWS documentation
 
+Enter the cloned directory:
+
+```
+cd amundsen/docs/installation-aws-ecs
```

Comment:
all the comment mentioned `~/<your-path-to-cloned-repo>/amundsenfrontendlibrary/docs/instalation-aws-ecs` , maybe we could change those comments (this doc was contributed back then we don't have the amundsen repo.

Reply:
@feng-tao totally agree